tornado/ioloop.py

```diff
@@ -161,6 +161,18 @@ def connection_ready(sock, fd, events):
 
     _current = threading.local()
 
+    @classmethod
+    def configure(cls, impl, **kwargs):
+        if asyncio is not None:
+            from tornado.platform.asyncio import BaseAsyncIOLoop
+
+            if isinstance(impl, (str, unicode_type)):
+                impl = import_object(impl)
+            if not issubclass(impl, BaseAsyncIOLoop):
```

Comment:
For the record, does that mean people will not be able to register their own custom IOLoop implementations? Is your long-term plan to remove IOLoop configuration altogether?

Reply:
Yes, we're getting rid of IOLoop customization. This PR is unfortunately rather disruptive but asyncio is inflexible in various ways that make it impossible to reach our (my) compatibility goals without going all in on asyncio on python 3. Custom IOLoops will remain available on python 2. For python 3, any customization of the event loop must go through asyncio instead of tornado.